app/services/NextUpdatesService.scala

```diff
@@ -159,6 +159,50 @@ class NextUpdatesService @Inject()(
 
     processingRes
   }
+
+  def getNextQuarterlyUpdateDueDate()(implicit hc: HeaderCarrier, mtdItUser: MtdItUser[_]): Future[Option[LocalDate]] = {
+    getOpenObligations().map {
+      case ObligationsModel(obligations) =>
+        val allQuarterlies = obligations
+          .flatMap(_.obligations)
+          .filter(o => o.obligationType == "Quarterly" && o.status == StatusOpen)
```

Comment:
You don't need to filter by `StatusOpen` here as the connector already calls only the open obligations via downstream.